resources/views/components/form-field.blade.php

```diff
@@ -1,33 +1,41 @@
-<div class="{{ ($inline ?? false) ? 'lg:flex' : '' }} {{ ($fullWidth ?? false) ? '' : '' }} mb-2">
-    @if($label ?? false)
-        <div class="{{ ($fullWidth ?? false) ? '' : (($inline ?? false) ? 'lg:w-36 lg:pr-3 lg:text-right' : '') }} pt-1 whitespace-nowrap {{ $errors && $errors->has($attribute ?? null) ? 'text-danger' : '' }}">
-            <label for="{{ $fieldId ?? $attribute ?? null }}">
-                {{ $label }} {{ !empty($required) ? '*' : '' }}
-            </label>
-        </div>
+@props([
+    'help' => null,
+    'hidden' => null,
+    'icon' => null,
+    'id' => null,
+    'inline' => false,
+    'label' => false,
+    'name' => null,
+    'prepend' => null,
```

Comment:
This seems to always be `null`, no matter what (unless there's a reference I haven't found). Is this a dead prop? One reason I'm wondering is because if this accepts a string, it could cause an a11y issue due to the new `aria-hidden="true"` attribute. May be best to remove.

Reply:
good point, it's not needed except for icon prepends. removed prop in https://github.com/RetroAchievements/RAWeb/pull/2247/commits/84f3880c0d8ec294cec6be61d7c05f705397eb77